Week3/homework/step2-2.js

```diff
@@ -2,19 +2,29 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+    const number = i;
+    const isNumberDivisibleBy3 = number % 3 === 0;
+    const isNumberDivisibleBy5 = number % 5 === 0;
+    const isNumberDivisibleBy15 = number % 15 === 0;
+    if (isNumberDivisibleBy3) {
+      console.log(threeCallback(number));
```

Comment:
It'd be better to just call the callback function - not to console log it.
Instead you'd put the console log within the function.

Reply:
ok